{{ cookiecutter.hosting_github_repo_name }}/.github/workflows/process-special-event.yml

```diff
@@ -0,0 +1,57 @@
+name: Process Special Event
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      event-details-json-string:
+        description: "Event details (see EventIngestionModel in cdp-backend) in JSON string form. The string must be free of any newline or tab characters."
+        required: true
+
+
+jobs:
+  process-events:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: 3.9
+
+    - name: Install Packages
+      run: |
+        sudo apt update
+        sudo apt-get install ffmpeg --fix-missing
+
+    - name: Install Python Dependencies
+      run: |
+        cd python/
+        pip install .
+
+    - name: Setup gcloud
+      uses: google-github-actions/setup-gcloud@master
+      with:
+        project_id: {{ cookiecutter.infrastructure_slug }}
+        service_account_key: {% raw %}${{ secrets.GOOGLE_CREDENTIALS }}{% endraw %}
+        export_default_credentials: true
```

Comment:
Can you make this change to the other pipeline workflows as well.

Reply:
Good call, added